### PR TITLE
Update rule for wrapping lines

### LINKED ISSUE #1490: Update eclipse rules wrt to wrapping lines
We should change

```
<setting id="org.eclipse.jdt.core.formatter.join_wrapped_lines" value="true"/>
```

To false so that fluent API calls that purposely put one API call per line aren't joined. This is an issue mostly with JDK stream() and reactive code.

### PR DESCRIPTION
**Pull request by ilayaperumalg**
- join_wrapped_lines to `false`

Resolves #1490

### CODE REVIEW

File: spring-cloud-dataflow-core/src/main/java/org/springframework/cloud/dataflow/core/StreamPropertyKeys.java [hunk 1]
```diff
@@ -23,7 +23,7 @@
  */
 public class StreamPropertyKeys {
 
-    // @formatter:off (not working)
+	// @formatter:off (not working)
     static final String PREFIX = "spring.cloud.stream.";
```
Comment (trisberg) [inline spring-cloud-dataflow-core/src/main/java/org/springframework/cloud/dataflow/core/StreamPropertyKeys.java]: Looks like we don't enforce spaces vs tabs, should/could we?

Comment (ilayaperumalg) [inline spring-cloud-dataflow-core/src/main/java/org/springframework/cloud/dataflow/core/StreamPropertyKeys.java]: No, this is an oversight. I don't know any ways of doing it yet :(

Comment (trisberg) [inline spring-cloud-dataflow-core/src/main/java/org/springframework/cloud/dataflow/core/StreamPropertyKeys.java]: do we know why the formatting isn't working on this class? seems like a simple class that should be easy to format

**Comment by trisberg** [comment]
Merged as commit e406a9877cd23cccd3224c3680fb529f1a2f405d

**Pull request closed without merging**